scripts/oxlint-hierarchical.sh

```
@@ -0,0 +1,221 @@
+#!/bin/bash
+
+# NeonPro OxLint Hierarchical Configuration Manager
+# Usage: ./scripts/oxlint-hierarchical.sh [command] [target]
+
+set -euo pipefail
+
+# Configuração
+PROJECT_ROOT=$(dirname $(dirname $(realpath $0)))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)